.eslintrc.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    jest: true,
+  },
+  extends: [
+    "plugin:react-hooks/recommended",
+    "standard",
+    "plugin:react/recommended",
+    "plugin:prettier/recommended",
+    "plugin:security/recommended",
+    "plugin:jsx-a11y/recommended",
+    "plugin:cypress/recommended",
+    "next",
+  ],
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 12,
+    sourceType: "module",
+  },
+  plugins: ["react"],
+  rules: {},
+  settings: {
+    react: {
+      version: "detect",
+    },
+  },
+};
```

Comment:
Added eslint for linting locally. Cypress linting is a nice one to have, as well as security, and a11y etc. I like StandardJS less config the better...

Jest added to the linting config so it doesn't blow up on tests